unit/src/MooseEnumTest.C

```diff
@@ -130,6 +130,10 @@ MooseEnumTest::testErrors()
     std::string msg(e.what());
     CPPUNIT_ASSERT( msg.find("Invalid option") != std::string::npos );
   }
+  catch(...)
+  {
+    // even though we throw std::runtime_exception, we are not able to catch it here on Mac OS X (it has some other type)
+  }
```

Comment:
I wonder if we'd be better off ifdef-ing this out instead. Having an empty catch on a real exception doesn't seem like the best plan. Either that or one of us can dig in and see what the right (if it exists) solution is to this problem.